src/schrodinger/sketcher/molviewer/scene.cpp

```diff
@@ -225,6 +226,20 @@ QRectF Scene::getInteractiveItemsBoundingRect(
     return bounding_rect;
 }
 
+bool Scene::isSimplifiedStereoAnnotationVisible() const
+{
+    return m_simplified_stereo_label->isVisible();
+}
```

Comment:
It looks like this is only called on line 237.  Is it worth having a separate method just for this?